### PR TITLE
Sparks leaderboard

### PR DESCRIPTION
**Pull request by dbhynds**
_Provide a general summary of your changes in the Title above._

## Proposed changes

Add a sparks label, and note if a user sparked the post. Note: we should probably merge the other tags-related PRs before checking out this one to make sure that there aren't any styling problems.

## Types of changes

_What types of changes does your code introduce? Put an `x` in all the boxes that apply:_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Unit Tests (only unit tests were added)
- [ ] Sprint Deployment

## How has this been tested?

Manual testing. This question should help with testing: http://questions.packback.local/questions/76D84C3F-84BF-726D-E653-A4F2459C320E?source=communityFeed

## Checklist

_*Pull Requester:* Put an `x` in the boxes that apply. If not applicable, leave the checkbox blank. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask the reviewer._

_*Reviewer:* If you see a blank checkbox where you  believe the action is applicable to the PR you are reviewing (and therefore should have an `x`), engage in a conversation (verbal or over Github comments) with the requester to understand their logic._

- [x] I have added the current sprint milestone to this PR.
- [x] I have attached this PR to all relevant cards.
- [x] Any dependent changes have been merged and released if needed.
- [x] My code follows the code style of this project.
- [x] I have added necessary documentation (if appropriate).
- [ ] I have added tests (unit and/or acceptance) that prove my fix is effective or that my feature works.
- [ ] My changes have manual deployment steps, and I have documented these in the current sprint's deployment card.
- [ ] I have checked off and verified that all items within the “Definition of Done” section on this work item have been met.

## Potential opportunities or challenges uncovered as a result of this change

- [ ] I have created inbox cards for any opportunities or future challenges that I was able to recognize in the process making this change.

## Further comments 

_Any other notes you wish to add. If this is a relatively large or complex change, it is recommended to explain why you chose the solution you did and what alternatives you considered, etc (though this can also be conveyed in-person)._

### CODE REVIEW

**Comment by dbhynds** [comment]
[Pete's Code Reviews](https://trello.com/c/5w0BoaeP/2201-petes-code-reviews)

**Comment by dbhynds** [comment]
[(1) Sparks label and text for showing how many users have sparked a post](https://trello.com/c/uLQvEOv6/2177-1-sparks-label-and-text-for-showing-how-many-users-have-sparked-a-post)

**Comment by dbhynds** [comment]
[(1) Update response and question object to include two horizontal lines in their "footer"](https://trello.com/c/YCzpV5M5/2184-1-update-response-and-question-object-to-include-two-horizontal-lines-in-their-footer)

File: assets/scss/atoms/_images.scss [hunk 1]
```diff
@@ -174,6 +174,20 @@ span.user-avatar {
...
+    background: url(../img/spark_inactive.svg) no-repeat;
+    background-position: center;
+}
+
+/// @example markup
+///   <div class="spark -active"></div>
+.spark.-active {
+    background: url(../img/spark_active.svg) no-repeat;
+    background-position: center;
+}
```
Comment (peteclodi) [inline assets/scss/atoms/_images.scss]: @db hynds If the reason for the positioning is the same as the `pin` background image sizing and positioning, then we should take the same tact of asking @jessicatenuta for a new image without the empty canvas.

File: assets/scss/placeholders/_qar-item-base.scss [hunk 2]
```diff
@@ -28,6 +28,11 @@
     .footer, .info {
         flex-wrap: wrap;
     }
+    .sparks-count {
+        font-weight: rem-calc(15);
```
Comment (peteclodi) [inline assets/scss/placeholders/_qar-item-base.scss]: @dbhynds I'm assuming you meant `font-size` and **not** `font-weight`

Comment (peteclodi) [inline assets/scss/placeholders/_qar-item-base.scss]: Also, is there any reason to make it's size 15 instead of 16?

**Pull request merged**